doc/geoipupdate.md

```diff
@@ -0,0 +1,87 @@
+# GEOIPUPDATE
+
+## NAME
+
+geoipupdate - GeoIP2, GeoLite2, and GeoIP Legacy Update Program
+
+## SYNOPSIS
+
+geoipupdate \[-Vvh\] \[-f license\_file\] \[-d target\_directory\]
```

Comment:
Giving multiple flags like this (-Vv) will not currently work unfortunately. They have to be -v -V. I wonder if we should change the behaviour.

Also `license_file` seems a strange name to me.

Reply:
Yeah, we might want to change the flag behavior for backcompat. We could use one of the argument modules that provides something more similar to getopt.

Re: `license_file` I agree. I'll fix that. It is from the old file.

Reply:
Re: argument parsing: I will add a ticket to do that.